README.md

```diff
@@ -354,6 +355,22 @@ _Mailing lists servers and mass mailing software - one message to many recipient
 - [SquirrelMail](http://squirrelmail.org) - Another browser-based IMAP client. ([Source Code](https://sourceforge.net/p/squirrelmail/code/HEAD/tree/)) `GPL-2.0` `PHP`
 - [WebMail Lite](http://www.afterlogic.org/webmail-lite) - Web-based IMAP Mail client. ([Source Code](https://github.com/afterlogic/webmail-lite)) `GPL-3.0` `PHP`
 
+#### Disposable email and forwarding services
+
+**[`^        back to top        ^`](#)**
+
+- [Ad Hoc Email Server (AHEM)](http://www.ahem.email) - An ad-hoc mail server. Create and use ad hoc disposable mailboxes for testing or any other purpose. http://www.ahem.email ([Demo](http://www.ahem.email), [Source Code](https://github.com/o4oren/Ad-Hoc-Email-Server)) `Angular/Nodejs` `MongoDB` `MIT`
```

Comment:
- Remove http://www.ahem.email from the description.  This is the same as site link.
- Remove the demo link it is also the same as the site link.
- Remove \`Mongodb` we are not listing databases.
- License should be LGPL-3.0